### ISSUE TITLE
[MangaJar] Connector not working

### ISSUE THREAD

**Issue by Mrnofish** [manga-download/hakuneko#3053]
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
Hakuneko only downloads the first page in each chapter

**To Reproduce**
1. Connector: MangaJar
2. Manga: F (Noboru Rokuda) https://mangajar.com/manga/f-noboru-rokuda/
3. Chapter: Download any chapter
4. Error: Failed to load resource: the server responded with a status of 404 (https://mangajar.com/manga/f-noboru-rokuda//chaptersList)

**HakuNeko (please complete the following information):**
 - Version 6.1.7

**Comment by Mrnofish** [comment]
@ronny1982 Excuse the most likely clueless comment, I'm still having the same issue with MangaJar (only the first page gets downloaded); I have:

1. Cleared the cache directory and let the wrapper download the whole web app again - no go.
2. Installed the latest web app for 6.1.7 manually as per instructions in the troubleshooting guide - still no go.
3. Did the unspeakable, and installed the web app from master - MangaJar.mjs gets replaced by the old version as soon as I start Hakuneko.

How do I get out of this conundrum?

Thanks in advance.

**Comment by 27136773** [comment]
What do you mean by MangaJar.mjs gets replaced?
HakuNeko updates itself on startup and if a new version is available it will be downloaded...

MangaJar works fine on my end with 6.1.7 testing random mangas and chapters

**Comment by Mrnofish** [comment]
6.1.7 does "update", in the sense it's downloading the web app from the Internet, but in all three cases it was downloading the old, non-working version of MangaJar.

I did find a workaround, in the meantime, which involved downloading the nightly portable and installing it to a different directory; the Nightly downloaded the updated web app with the fixed MangaJar and I was able to download the manga just fine.

6.1.7, on the other hand, is a total non-starter for me. I'm really at a loss what's going wrong here.
